MANIFEST.in

```diff
@@ -0,0 +1,15 @@
+include CHANGES.txt
+include LICENSE.txt
+include MANIFEST.in
+include README.rst
+include image_LICENSE.txt
+include image_LICENSE_Eclipse.txt
+include image_LICENSE_Nuvola.txt
+include TODO.txt
+graft docs
```

Comment:
Yes.

Whether or not they should be there still (after 7 years!) when the same content is available online is another question.  But not one for this PR.